lib/kasket/visitor.rb

```diff
@@ -144,6 +144,7 @@ def visit_FalseClass(_node)
     end
 
     def quoted(node)
+      return node if node.is_a?(String)
```

Comment:
humm.. why?

Reply:
forget it. half sleeping yet.

Reply:
We end up generating invalid SQL where strings are quoted twice

Reply:
Hmm, it looks like quoting a string works identically in Rails 3 and 4:
```rb
# Loading development environment (Rails 3.2.22.4)
ActiveRecord::Base.connection.quote("foo")
#=> "'foo'"

# Loading development environment (Rails 4.2.10)
ActiveRecord::Base.connection.quote("foo")
#=> "'foo'"
```